includes/integrations/class-scaip.php

```diff
@@ -146,6 +149,25 @@ public static function create_placement_action( $atts ) {
 		do_action( sprintf( self::HOOK_NAME, $atts['number'] ), $atts );
 	}
 
+	/**
+	 * Setup Newspack Ads defaults on activation.
+	 */
+	public static function register_defaults() {
```

Comment:
I can see the settings in the wizard, but the SCAIP placements are not inserted in an article. How to debug that?

<img width="891" alt="image" src="https://user-images.githubusercontent.com/7383192/171189865-8e434df4-1fd6-4d56-9b68-dc79fc7a07c8.png">